src/core/sdl/render_sdl.cpp

```diff
@@ -398,13 +396,20 @@ void RenderSDL::loadTexture(StdPicture &target, uint32_t width, uint32_t height,
     if(!texture)
     {
         pLogWarning("Render SDL: Failed to load texture! (%s)", SDL_GetError());
-        target.d.clear();
         target.inited = false;
         return;
     }
 
     target.d.texture = texture;
-    m_textureBank.insert(texture);
+
+    target.d.nOfColors = GL_RGBA;
+    target.d.format = GL_BGRA;
+
+    target.d.w_scale = static_cast<float>(width) / target.w;
+    target.d.h_scale = static_cast<float>(height) / target.h;
+
+    m_loadedPictures.insert(&target);
+    D_pLogDebug("RenderSDL: loading texture at %p, new texture count %d...", &tx, (int)m_loadedPictures.size());
```

Comment:
There is build failure because of `tx` thing that caome from nowhere :eyes:

![Снимок экрана от 2023-04-17 01-14-49](https://user-images.githubusercontent.com/6751442/232345638-08be3c54-844b-4519-b46d-22a40213e7a3.png)

Reply:
Oops, I must have converted it to `D_` before refactoring the code a final time. Thanks for the fix :+1:

Reply:
The `D_` means works when it's a debug build, so, it's fine when it's printed at debug-only builds.